src/Assets/ugui-mvvm/CommandBinding.cs

```diff
@@ -50,21 +54,21 @@ public object Parameter
         INPCBinding.PropertyPath _vmProp;
         private ICommand _command;
 
-        public INPCBinding.ComponentPath ViewModel
+        public INPCBinding.ComponentPath Source
         {
             get
             {
-                return _viewModel;
+                return _source;
             }
         }
 
         void Reset()
         {
             var context = gameObject.GetComponentInParent(typeof(DataContext)) as DataContext;
             if (context != null)
-                _viewModel = new INPCBinding.ComponentPath { Component = context };
+                _source = new INPCBinding.ComponentPath { Component = context };
```

Comment:
Not your change, but a bunch of these if statements lack braces.  Consider adding braces to all of them that you've touched.

Reply:
Visual Studio can do a bulk fix to add braces where they are missing. I will submit this as a separate PR.